src/app/app.component.ts

```diff
@@ -268,6 +270,13 @@ export class AppComponent {
     this._fullscreen.addFullscreenListener();
   }
 
+  /**
+   * Determines whether to display role prompt
+   */
+  private initializeRolePrompt(shouldPromptForRole: boolean): void {
+    this.showRolePrompt = this._auth.shouldPromptForRole() && shouldPromptForRole
```

Comment:
Ultra nit but these feel a bit backwards to me, assuming this would short circuit to false if the `shouldPromptForRole` flag value is false.

Reply:
@michael-iden like this? 
`this.showRolePrompt = shouldPromptForRole && this._auth.shouldPromptForRole()`

Reply:
You got it. sry I should've added the suggestion from the start
```suggestion
    this.showRolePrompt = shouldPromptForRole && this._auth.shouldPromptForRole()
```

Reply:
No worries! Changed.